docs/introduction/what_is_jellyfish.md

```diff
@@ -14,14 +14,18 @@ In general, there are no limitations.
 See [Example Scenarios](example_scenarios.md) for more information.
 
 Jellyfish comes with:
-* built-in authentication
-* client and server SDKs
-* support for multiple media protocols
-* docker images for multiple architectures
+* [built-in authentication](../getting_started/authentication.md)
+* [client and server SDKs](../getting_started/sdks.md)
+* [server side notifications](../getting_started/notifications.md)
+* [support for multiple media protocols](#supported-protocols)
+* [built-in clustering](../cluster.md)
+* [tutorials](../tutorials/dashboard.mdx)
```

Comment:
Shouldn't we have a page with `tutorials` where we would describe all available tutorials?

Reply:
I don't think so. They are listed when someone goes to the `Tutorials` section. We would have to duplicate content